setup.py

```diff
@@ -9,6 +9,7 @@
     'pycodestyle',
     'typing;python_version<"3.5"',
     'typing-extensions',
+    'mypy==0.761'
```

Comment:
I don't think we want installing this to require pulling in `mypy`, especially with a fixed version (which might disagree with what the third-party is using in their virtualenv).  Maybe could be in a "development requirements".